src/commands/createNewProject.ts

```diff
@@ -0,0 +1,260 @@
+    }
+    return null;
+}
+
+/**
+ * Prompts user to select a workspace folder
+ */
+async function selectWorkspaceFolder(): Promise<string | undefined> {
+    const options: vscode.OpenDialogOptions = {
+        canSelectFiles: false,
+        canSelectFolders: true,
+        canSelectMany: false,
+        title: 'Select folder for new Roku project'
+    };
+
+    const folderUri = await vscode.window.showOpenDialog(options);
+    return folderUri?.[0]?.fsPath;
+}
+
+/**
+ * Creates the basic project folder structure
+ */
+async function createProjectStructure(projectPath: string) {
+    const folders = [
+        '',
+        'source',
+        'components',
+        'images'
+    ];
+
+    for (const folder of folders) {
+        await fs.promises.mkdir(path.join(projectPath, folder), { recursive: true });
+    }
+}
+
+/**
+ * Copies template images from the extension's assets to the new project
+ */
+async function copyTemplateImages(extensionPath: string, projectPath: string) {
+    try {
+        const imageFiles = [
+            { src: 'channel-poster_hd.png', dest: 'channel-poster_hd.png' },
+            { src: 'channel-poster_sd.png', dest: 'channel-poster_sd.png' },
+            { src: 'splash-screen_sd.jpg', dest: 'splash-screen_sd.jpg' },
+            { src: 'splash-screen_hd.jpg', dest: 'splash-screen_hd.jpg' }
+        ];
+
+        // The source images are in the 'images' directory of the extension
+        const sourceImagesPath = path.join(extensionPath, 'images');
+        const targetImagesPath = path.join(projectPath, 'images');
+
+        vscode.window.showInformationMessage(`Copying template images from ${sourceImagesPath} to ${targetImagesPath}`);
+
+        for (const image of imageFiles) {
+            try {
+                const sourcePath = path.join(sourceImagesPath, image.src);
+                const targetPath = path.join(targetImagesPath, image.dest);
+                
+                vscode.window.showInformationMessage(`Copying ${sourcePath} to ${targetPath}`);
+                
+                await fs.promises.copyFile(sourcePath, targetPath);
+            } catch (error) {
+                vscode.window.showWarningMessage(`Warning: Failed to copy template image ${image.src}: ${error.message}`);
+            }
+        }
+    } catch (error) {
+        throw new Error(`Failed to copy template images: ${error.message}`);
+    }
+}
+
+
+/**
+ * Creates all necessary project files with template content
+ */
+async function createProjectFiles(projectPath: string, projectName: string) {
+    // Create manifest file
+    const manifestContent = `
+title=${projectName}
+major_version=1
+minor_version=0
+build_version=0
+mm_icon_focus_hd=pkg:/images/channel-poster_hd.png
+mm_icon_focus_sd=pkg:/images/channel-poster_sd.png
+
+splash_screen_sd=pkg:/images/splash-screen_sd.jpg
+splash_screen_hd=pkg:/images/splash-screen_hd.jpg
+
+ui_resolutions=hd
```

Comment:
```suggestion
ui_resolutions=fhd
```

Most developers design in FHD I think.

Reply:
Yeah, I just don't have a FHD poster image right now. Let me find that